Umbraco-Cloud/Troubleshooting/Deployments/index.md

```diff
@@ -36,3 +36,76 @@ The most common Content [Transfer](../../Deployment/Content-Transfer) / [Restore
+	2022-11-02T08:51:16.1509492Z,https://aka.ms/dotnet-core-applaunch?framework=Microsoft.NETCore.App&framework_version=7.0.0-rc.1.22426.10&arch=x86&rid=win10-x86,,1
+	2022-11-02T08:51:16.1821221Z,You must install or update .NET to run this application.\r\n\r\nApp: C:\Program Files (x86)\dotnet\sdk\7.0.100-rc.1.22431.12\dotnet.dll\r\nArchitecture: x86\r\nFramework: 'Microsoft.NETCore.App'&comma; version '7.0.0-rc.1.22426.10' (x86)\r\n.NET location: C:\Program Files (x86)\dotnet\\r\n\r\nThe following frameworks were found:\r\n  1.0.16 at [C:\Program Files (x86)\dotnet\shared\Microsoft.NETCore.App]\r\n  1.1.13 at [C:\Program Files (x86)\dotnet\shared\Microsoft.NETCore.App]\r\n  2.0.9 at [C:\Program Files (x86)\dotnet\shared\Microsoft.NETCore.App]\r\n  2.1.30 at [C:\Program Files (x86)\dotnet\shared\Microsoft.NETCore.App]\r\n  2.2.14 at [C:\Program Files (x86)\dotnet\shared\Microsoft.NETCore.App]\r\n  3.0.3 at [C:\Program Files (x86)\dotnet\shared\Microsoft.NETCore.App]\r\n  3.1.28 at [C:\Program Files (x86)\dotnet\shared\Microsoft.NETCore.App]\r\n  3.1.29 at [C:\Program Files (x86)\dotnet\shared\Microsoft.NETCore.App]\r\n  5.0.15 at [C:\Program Files (x86)\dotnet\shared\Microsoft.NETCore.App]\r\n  5.0.17 at [C:\Program Files (x86)\dotnet\shared\Microsoft.NETCore.App]\r\n  6.0.8 at [C:\Program Files (x86)\dotnet\shared\Microsoft.NETCore.App]\r\n  6.0.9 at [C:\Program Files (x86)\dotnet\shared\Microsoft.NETCore.App]\r\n\r\nLearn about framework resolution:\r\nhttps://aka.ms/dotnet/app-launch-failed\r\n\r\nTo install missing framework&comma; download:\r\nhttps://aka.ms/dotnet-core-applaunch?framework=Microsoft.NETCore.App&framework_version=7.0.0-rc.1.22426.10&arch=x86&rid=win10-x86\r\nC:\Program Files (x86)\SiteExtensions\Kudu\98.40824.5897\bin\Scripts\starter.cmd C:\home\SiteExtensions\Umbraco.Cloud.Deployment.SiteExtension.Artifacts.Core\deploy.cmd,,2
+2022-11-02T08:51:16.9321330Z,Deployment Failed.,133sa799-q231-c92a-a244-afa18e1c2b1f,0
+```
+
+Step 1.
+Clone down your Development environment. If you only have a Live environment, then proceed with that  
+
+Step 2.
+Add a global.json to root of the project. Your structure should look like this:
+```html
+.git
+src/
+.dockerignore
+.editorconfig
+.gitattributes
+.gitignore
+.umbraco
+global.json
+NuGet.config
+Readme.md
+```
+
+Step 3.
+Open the global.json file and add the following:
+```json
+{
+    "sdk": {
+      "version": "6.0.401"
+    }
+}
+```
+
+And save the file.
+
+Step 4.
+Add, commit the file and push it to the env.
+
+This will force the Azure build service to target version 6.0.401 of the .net SDK for your project and your env will function again.
+You will not need to add the same file to your other environments as source control will take care of adding it in when you deploy it between your development, staging or live environments.
+
+When you have done the above steps you should no longer see deployment issues with .net 7-preview or rc.
```

Comment:
```suggestion
1. Clone down your Development environment. If you only have a Live environment, then proceed with that  

2. Add a global.json to root of the project. Your structure should look like this:

```html
.git
src/
.dockerignore
.editorconfig
.gitattributes
.gitignore
.umbraco
global.json
NuGet.config
Readme.md
```

3. Open the global.json file and add the following:
```json
{
    "sdk": {
      "version": "6.0.401"
    }
}
```

And save the file.

4. Add, commit the file, and push it to the env.
This will force the Azure to build a service to target version 6.0.401 of the .net SDK for your project and your env will function again.
You will not need to add the same file to your other environments as source control will take care of adding it in when you deploy it between your development, staging, or live environments.

When you have done the above steps you should no longer see deployment issues with .net 7-preview or RC.
```